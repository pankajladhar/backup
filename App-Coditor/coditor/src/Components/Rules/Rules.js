import React from "react";

const rulesData = [
  "1. some basic challenge rules and Instructions",
  "2. some basic challenge rules and Instructions",
  "3. some basic challenge rules and Instructions",
  "4. some basic challenge rules and Instructions",
  "5. some basic challenge rules and Instructions",
  "6. some basic challenge rules and Instructions"
];

const Rules = () => {
  return (
    <div className=" flex h-screen flex-col p-12">
      <h3 className="text-4xl mb-5 font-display text-indigo-900 font-extrabold">
        Instructions
      </h3>
      <ul>
        {rulesData.map((rule, index) => {
          return <li key={`rule-${index}`}>{rule}</li>;
        })}
      </ul>
    </div>
  );
};

export default Rules;
