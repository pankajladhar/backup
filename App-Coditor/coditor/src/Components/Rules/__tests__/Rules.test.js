import React from "react";
import renderer from "react-test-renderer";
import Rules from "./../Rules";

it("should render properly", () => {
  const tree = renderer.create(<Rules />);
  expect(tree.toJSON()).toMatchSnapshot();
});
