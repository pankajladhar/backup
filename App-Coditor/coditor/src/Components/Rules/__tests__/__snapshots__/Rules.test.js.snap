// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render properly 1`] = `
<div
  className=" flex h-screen flex-col p-12"
>
  <h3
    className="text-4xl mb-5 font-display text-indigo-900 font-extrabold"
  >
    Instructions
  </h3>
  <ul>
    <li>
      1. some basic challenge rules and Instructions
    </li>
    <li>
      2. some basic challenge rules and Instructions
    </li>
    <li>
      3. some basic challenge rules and Instructions
    </li>
    <li>
      4. some basic challenge rules and Instructions
    </li>
    <li>
      5. some basic challenge rules and Instructions
    </li>
    <li>
      6. some basic challenge rules and Instructions
    </li>
  </ul>
</div>
`;
